Chek HomeTask

DanyaKorolkov opened the pull request.

DanyaKorolkov committed:
Add files via upload

MathPracticum reviewed (changes requested):
нужно переписать 1-4 без циклов и доделать кластеризацию

DanyaKorolkov committed:
VERSION 2.0

Так-с, я добавлял этот же файл (за исключением незначительных изменений) давно. В нём я учёл все ваши пометки. Кажется, я не сделал тогда Pull Request вновь, так что залью файл заново (чуть-чуть подправив его)

korolkovDA commented:
А, pull request делать и не надо, достаточно просто залить новый файл - удобно)

MathPracticum commented:
в PR нужно оставить только один файл, задача на k-means недоделана

DanyaKorolkov committed:
Delete HomeTask_2.ipynb

DanyaKorolkov committed:
Update HomeTask 2.ipynb
